src/main/java/github/weichware10/analyse/Client.java

```diff
@@ -1,6 +1,6 @@
 package github.weichware10.analyse;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.